data/tlg0014/tlg046/tlg0014.tlg046.perseus-grc2.xml

```diff
@@ -29,13 +29,16 @@
 				<biblStruct>
 					<monogr>
 						<author>Demosthenes</author>
-						<title>Demosthenis. Orationes. Vol. III. ed W. Rennie</title>
+						<title xml:lang="lat">Orationes</title>
+						<editor>W. Rennie</editor>
```

Comment:
I think the names were being spelled out?

Reply:
hi @lcerrato I was going to do a final pass at the end, but I can also do one first, since it is of course different editors!